src/lib.rs

```diff
@@ -198,6 +203,27 @@ fn build_poi(row: Row) -> Option<Poi> {
         return None;
     }
 
+    let properties = build_poi_properties(&row, &name).unwrap_or(vec![]);
+
+    let weight_wikidata = properties
+        .iter()
+        .find(|p| &p.key == "wikidata")
+        .map_or(0., |_p| WEIGHT_TAG_WIKIDATA);
```

Comment:
This weight needs to be normalized too.   
This could be caught with a test case ;)

Reply:
woups, my bad ^^
yup for the tests